starter-code/apple-pie-starter-code/css/main.css

```diff
@@ -0,0 +1,51 @@
+* {
+    box-sizing: border-box;
+}
+
+body {
+    font-family: Verdana;
+}
+
+ol {
+    counter-reset: list;
+  }
+  ol > li {
+    list-style: none;
+  }
+  ol > li:before {
+    content: counter(list) ") ";
+    counter-increment: list;
+  }
+
+  img {
+      width: 300px;
+      display: block;
+      margin: auto;
+  }
+
+#banner {
+    background-image: url("./images/apple-pie.jpg");
```

Comment:
The background is not displaying because the route is wrong. The folder 'images' is outside the folder 'css'. Try with:
```
background-image: url("../images/apple-pie.jpg");
```